### DISCUSSION TITLE
Website not updating

### DISCUSSION THREAD

**Discussion by cafemarsbk** [tinacms/tinacms#5286] [Help]
Hi!

I made a slew of changes through the TinaCloud front-end editor (toggling sections on/off, adding images, etc.) about 2 hours ago. None of them, despite having been saved and continuing to show in the editor, are displaying on my actual website. Does anyone know if something is down? Anyone having the same issue?

**Comment by ncn-ssw** [comment]
CC @brookjeynes-ssw 

Hi @cafemarsbk,

TinaCloud should be operating correctly at this time. What is your Client ID?

**Comment by cafemarsbk** [comment]
Hi!

Thanks for the follow up! My client ID is

34c283a2-78a1-472a-8d66-8bc8bd0c7ac0

The saved changes are still not reflected on the site.

Thanks so much,
Paul

**Reply by kldavis4** [reply]
Are the changes in GitHub?

**Comment by cafemarsbk** [comment]
They’re in the Tina editor, it’s not code, just making content changes.

**Comment by kldavis4** [comment]
Tina is git based - when you make changes in the editor, it updates the
associated github repository

**Comment by cafemarsbk** [comment]
Ok, then the changes are in GitHub.

**Comment by kldavis4** [comment]
How are you deploying the site? Is it entirely static?

**Comment by cafemarsbk** [comment]
I’m sorry, I don’t know what you mean by that. I didn’t build the site, I’m
just the end user updating content for my restaurant. But this has never
been an issue in the last 1.5 years and I made weekly changes to the site.

**Comment by kldavis4** [comment]
Ok, you are probably going to be better off reaching out to the builder of
your site first. If they have technical questions about Tina, you can
direct them to this issue.

**Comment by cafemarsbk** [comment]
Hi,

Unfortunately, she left us in the lurch and disappeared after migrating to
Tina. Like I said, these are changes that are made regularly. Nothing has
happened in the code or build of the site. Might there be some sort of
update needed? I would imagine not given that it’s all web-based and
auto-updated.

This is an issue of Tina cloud not talking to our hosting provider.

**Reply by kldavis4** [reply]
Tina Cloud does not deploy your site. If you want to invite my github user (kldavis4) to your github repository, I may be able to help you determine what the issue might be.

**Comment by cafemarsbk** [comment]
Hi Kelly!

Thank you, I totally missed this when you sent. I just sent an invite.

Thanks so much,
Paul

**Reply by kldavis4** [reply]
After looking at your repository, it appears that the site is deployed using netlify. You'll need to login there and check to the deployment logs to see if you have errors there.

**Comment by cafemarsbk** [comment]
Hi,

I really appreciate your help and patience on this. I took your advice and
found that all my updates had failed in the deployment log on netifly, but
had no reason in the log for the failures. I went on vacation for 4 days,
and was about to go back in for a deeper dive, but checked my site first to
see if anything had happened. All the updates had gone through and the site
is about updating within 2-3 minutes from changes made in the tina editor.
Weird, but I'll take it!

Thanks so much,
Paul
